src/main/java/frc/trigon/robot/misc/objectdetectioncamera/SimulationObjectDetectionCameraIO.java

```diff
@@ -68,7 +68,7 @@ private void updateObjectEjection() {
         if (heldObject.length == 0 || !isEjecting() || isDelayingEjection)
             return;
         isDelayingEjection = true;
-        Commands.getDelayedCommand(0.04, () -> {
+        GeneralCommands.getDelayedCommand(0.04, () -> {
```

Comment:
I would make a var for this

Reply:
That's not stuff that I did. I just changed the name of the class from `Commands` to `GeneralCommands`